pages/blueprints/index.js

```diff
@@ -114,6 +122,24 @@ class BlueprintsPage extends React.Component {
     e.stopPropagation();
   }
 
+  startService(e) {
+    if (!e || e.button !== 0) return;
+
+    let argv;
+    if (this.state.enableService) argv = ["systemctl", "enable", "--now", "lorax-composer.socket"];
+    else argv = ["systemctl", "start", "lorax-composer.socket"];
```

Comment:
Please put the bodies of the `if` and `else` keywords on separate, indented lines.